### DISCUSSION TITLE
css: Unknown rule @tailwind @apply

### DISCUSSION THREAD

**Discussion by cutbypham** [LazyVim/LazyVim#520] [General]
> nvim/lua/plugins/LSP.lua
```lua
return {
  {
    "neovim/nvim-lspconfig",
    opts = {
      servers = {
        cssls = { settings = { css = { lint = { unknownAtRules = 'ignore', }, }, }, },
      },
    },
  },
}
```

**Comment by marekvospel** [comment]
Note: the same option works for the `svelte` and `volar` language servers! (under the same path - `settings.css.lint.unknownAtRules`)
This way you can get rid of this warning even in vue/svelte!

**Comment by VongoSanDi** [comment]
It don't work for me, created in ~/.config/nvim/lua/plugins/lsp.lua but nothing changed, i'm using tailwindv4, vuejsv3 and vitev6.

**Reply by valeriancure** [reply]
Got it working by targeting `vue_ls` :

````lua
return {
  {
    "neovim/nvim-lspconfig",
    opts = {
      servers = {
        vue_ls = {
          settings = {
            css = {
              validate = true,
              lint = {
                unknownAtRules = "ignore",
              },
            },
          },
        },
       ...
````

**Comment by harshv5094** [comment]
<img width="1366" height="768" alt="image" src="https://github.com/user-attachments/assets/f8c2ab76-65f7-474e-a27e-804338bcfb8b" />

I have a question, can I also remove this warning?
